modules/apps/static/portal-file-install/portal-file-install-impl/src/main/java/com/liferay/portal/file/install/internal/Scanner.java

```diff
@@ -217,6 +222,18 @@ private Set<File> _processFiles(boolean reportImmediately, File[] list) {
 				continue;
 			}
 
+			String path = file.getParent();
+			String extention = FilenameUtils.getExtension(file.getName());
+			List<String> extentions = new ArrayList<>(
+				Arrays.asList("config", "cfg"));
+
+			if (extentions.contains(extention) &&
```

Comment:
@locpham97 @thienquach87 The `Scanner` is a universal part of our FileInstall implementation which should not contain type-specific logic. If you think FileInstall should only install config files from the configs dir, you should add the logic to `ConfigurationFileInstaller.canTransformURL(File)` method. If it doesn't work, please let me know.